platform/i18n/src/locales/en-US/AboutModal.json

```diff
@@ -1,14 +1,14 @@
 {
   "Browser": "Browser",
-  "Build Number": "Build Number",
-  "Latest Master Commits": "Latest Master Commits",
+  "Build number": "Build Number",
+  "Last master commits": "Latest Master Commits",
   "More details": "More details",
   "Name": "Name",
   "OHIF Viewer - About": "OHIF Viewer - About",
```

Comment:
I think with the changes in this PR, this should be...

`"About OHIF Viewer":"About OHIF Viewer"`